lib/registries/addon/components/license-viewer/styles.scss

```diff
@@ -2,3 +2,13 @@
     padding: 0;
     border: 0;
 }
+
+.LicenseName {
+    text-overflow: ellipsis;
+    white-space: normal;
```

Comment:
I took a look at this locally setting up mirage so that it would have a long enough license, and it looks like this line is the key. Everything else doesn't seem to do anything but this, which fixes it properly (at least in safari and chrome). But since this is technically a button, the text is centered, so you need to add a `text-align: left;` to this as well to make it come out perfectly. 

<img width="301" alt="Screen Shot 2021-05-03 at 10 59 29 AM" src="https://user-images.githubusercontent.com/6599111/116894095-a6e57d80-abff-11eb-8802-89558047b66b.png">

Reply:
Of course and thank you. In my testing I found that removing the white-space: normal property and the Mozilla property to word-break as well was ok. When I go to remove any of the other properties however it will result in breaking changes, at least in the online editor. There may be important differences between the two, but remember a similar result when testing locally on localhost:4200 using Docker as a backend. If possible, could look at this further during our meeting today? 

![image](https://user-images.githubusercontent.com/82047646/116918678-37cb5180-ac1e-11eb-8870-9f1f877751cc.png)

![image](https://user-images.githubusercontent.com/82047646/116919110-d2c42b80-ac1e-11eb-8f07-cd74917b2375.png)

![image](https://user-images.githubusercontent.com/82047646/116918985-a27c8d00-ac1e-11eb-95be-f50f225b6830.png)

Reply:
Hmm, I'm not having the same experience. If I turn off the `white-space` and turn everything else back on, or if I match what you have up there, I'm getting the overflow just running out of the box. The only thing that works on my setup is to have `white-space: normal;` set.

<img width="531" alt="Screen Shot 2021-05-03 at 3 16 45 PM" src="https://user-images.githubusercontent.com/6599111/116922136-ec677200-ac22-11eb-8d37-b3bc8a3212fe.png">

<img width="354" alt="Screen Shot 2021-05-03 at 3 18 10 PM" src="https://user-images.githubusercontent.com/6599111/116922171-f7ba9d80-ac22-11eb-98ed-1120d0f3989c.png">